### PR TITLE
Milestone 2: Create pull request.

### PR DESCRIPTION
**Pull request by abelagza**
# Created pull request.

In this milestone I :

- Added linters
- Added Test.md file
- Added README file
- Added HTML file
- Added CSS file
- Linked CSS file to HTML

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (gemmen29) [inline .gitignore:1]: - Kindly add `node_modules` in the .gitignore file as it's a requirement in this project.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+# Live Demo 
+Not available
+
+# Getting Started
+  To view the content of this project you will be required to install a text editor and the Web Browser of your choice.
+
+# Authors
+  Github: @abelagza
+  Twitter: @AbelAgza
+  LinkedIn: abelagza
```
Comment (gemmen29) [inline README.md:18]: Please change those with actual links. You can do that by doing that [@abelagza](actual link)

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @abelagza, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- Please add those files in linters repo [.hintrc](https://github.com/microverseinc/linters-config/blob/master/html-css/.hintrc) and [.stylelintrc.json](https://github.com/microverseinc/linters-config/blob/master/html-css/.stylelintrc.json).

_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Donard97 — APPROVED** [review]
## Hi @abelagza , ✋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 🎉

giphy (9)

**Cheers and Happy coding!👏👏👏**

## Optional suggestions

- [ ] Please add node_modules in gitignore file
- [ ] Kindly update README file using Microverse template

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**